model/src/main/kotlin/OrtResult.kt

```diff
@@ -395,7 +402,7 @@ data class OrtResult(
         }
 
         return if (omitResolved) {
-            val resolutions = getRepositoryConfigResolutions().ruleViolations
+            val resolutions = getResolutions().ruleViolations
```

Comment:
How about crafting this commit slightly different by only introducing `getResolutions()` at first, and then taking it into use in all relevant places at once?

That would mean to move this line to the next commit, and to squash the related commits following the next commit also into the next commit.